keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/artifact/ImageHandler.kt

```diff
@@ -88,7 +88,7 @@ class ImageHandler(
   private suspend fun DebianArtifact.findLatestArtifactVersion(): String {
     try {
       val knownVersion = repository
-        .artifactVersions(this)
+        .artifactVersions(this, 1)
```

Comment:
This was unnecessarily retrieving multiple versions when we know we only want the first one.